timetable/templatetags/timetable_tags.py

```diff
@@ -33,15 +35,22 @@ def for_event(weeks, event):
 
 
 @register.simple_tag
-def room_link(room):
-    room = room.strip().translate(affix_map)
-    # print(room)
-    if len(room) == 0:
-        return None
-    try:
-        roomlink = RoomLink.objects.get(room__iexact=room)
-    except (RoomLink.DoesNotExist, RoomLink.MultipleObjectsReturned):
-        roomlink = RoomLink.objects.filter(room__icontains=room).first()
-    if roomlink is None:
-        return None
-    return roomlink.url
+def room_link(r, _class):
+    # Remove all special characters from affixes.
+    rooms = r.split(",")
+    rooms = [i.strip().translate(affix_map) for i in rooms]
+    results = []
+    for room in rooms:
+        roomlink = None
+        # print(room)
+        if len(room) > 0:
+            try:
+                roomlink = RoomLink.objects.get(room__iexact=room)
+            except (RoomLink.DoesNotExist, RoomLink.MultipleObjectsReturned):
+                roomlink = RoomLink.objects.filter(room__icontains=room).first()
+        if roomlink is None:
+            results.append(room)
+        else:
+            results.append(format_html('<a target="_blank" rel="noopener noreferrer" href="{}" class="{}">{}</a>',
+                roomlink.url, _class, room))
```

Comment:
You unfortunately can't use `room` here, as its had any suffixes it may have had stripped off.

Reply:
Also, as you've lost spaces (due to strip), `R0.03, R0.04` and `R0.03,R0.04` are rendered the same, but aren't combined together.